source/administration/bulk-export-data.rst

```diff
@@ -0,0 +1,310 @@
+Bulk export data
+=================
+At this time, the export supports attributes of the objects listed below.   Configuration for exporting specific areas of the server, exporting additional types of posts, reactions, file attachments, webhooks and bot messages are not yet supported. Deleted objects are not yet supported as well.  For requests to add additional attributes or objects to our exporter, please add a feature request on our `feature idea formum <https://mattermost.uservoice.com/forums/306457-general>`_.  
+
+Version object
```

Comment:
Sorry, one more thing here. Can I suggest a little text under this title that explains that:

```
All Mattermost Bulk Export data files will begin with a `Version` object as the first line of the file. This indicates the version of the Mattermost Bulk Import file format with which the exported data is compatible.
```

Reply:
Did you decide against adding this, or was it missed accidentally?

Reply:
I had a pending edit that I didn't get in.  Added now.